package-examples/ghost/ghost/deployment-example-ghost.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: example-ghost
+  namespace: example
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      annotations:
+        checksum/secrets: 711677f8a87ff3df1d09863858613b5dbecb3f6b8ea51a8c37b40762aff66a20
+    spec:
+      affinity:
+        podAffinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - podAffinityTerm:
+                namespaces:
+                  - "example"
```

Comment:
This is interesting for `set-namespace`'s function perspective.

Reply:
Yes, so as the namespaceSelector for podAffinityTerm.